core/app/models/spree/order_shipping.rb

```diff
@@ -70,7 +70,10 @@ def ship(inventory_units:, stock_location:, address:, shipping_method:,
       # TODO: make OrderShipping#ship_shipment call Shipment#ship! rather than
       # having Shipment#ship! call OrderShipping#ship_shipment. We only really
       # need this `update_columns` for the specs, until we make that change.
-      shipment.update_columns(state: 'shipped', shipped_at: Time.current)
```

Comment:
The comment above state that we need this `update_columns` for specs only. Do you think it's still valid?